schema/catvrs/catvrs-source.yaml

```diff
@@ -13,296 +13,254 @@ namespaces:
       label:
         extends: label
         description: >-
-          A primary label for the categorical variation. This required property should provide a 
-          short and descriptive textual representation of the concept.
-      description:
-        extends: description
+            A primary label for the categorical variation. This required property should provide a short and descriptive textual representation of the concept.
+      constraints:
+        type: array
+        items: #do we have to have a value here for JSON schema or just require it to be non-empty?
+      extensions:    # Are extensions still being used in current VRS? 
+        type: array
+        ordered: true
+        items:
+          $refCurie: gks.core:Extension
+        description: 
+          A list of extensions to the entity. Extensions are not expected to be natively understood, but may be used for pre-negotiated exchange of message attributes between systems.
+
+      mappings:
+        type: array
+        description: 
+          allowed mapping operations for this context
+          items: [$ref: coding] # ditto not sure what the items consist of
+      members:
+        type: array
+        ordered: false
         description: >-
-          A textual description of the domain of variation that should match the categorical 
-          variation entity.
+          A non-exhaustive list of VRS variation contexts that satisfy the constraints of
+          this categorical variant.
+        items:
+          oneOf:
+            - $refCurie: vrs:Variation
+            - $refCurie: gks.common:IRI
     required:
-      - label
-
-
-  # test_parent_A:
-  #   maturity: draft
-  #   heritable properties:
-  #     type:
-  #       type: string
-  #     prop1:
-  #       type: string
-  #       const: "property from the test parent class"
-  #       description: >-
-  #         Must be "property from the test parent class"
-  #   description: >-
-  #     test parent ID.  MUST be "test_parent"
-  #   # heritable_required: ["type"]
-
-  # test_child_B:
-  #   inherits: test_parent_A
-  #   # type: object
-  #   description: >-
-  #     Child_B, a child of test_parent_A
-  #   heritable properties:
-  #     prop2:
-  #       type: string
-  #       const: "property from the test_child_B"
-  #       description: >-
-  #         Must be "property from the test_child_B"
-    
-  # test_child_C:
-  #   inherits: test_parent_A
-  #   description: >-
-  #     Child_C, a child of test_parent_A
-  #   heritable properties:
-  #     prop3:
-  #       type: string
-  #       const: "property from the test_child_C"
-  #       description: >-
-  #         Must be "property from the test_child_C"
-
-
-  # test_grandchild_D:
-  #   inherits: test_child_B, test_child_C
-  #   # inherits: test_child_C
-  #   description: >-
-  #     Child_C, a child of test_child_B and test_child_D
+      -id
+      -label
```

Comment:
Label is required? What if a categorical variation is not labeled, i.e. is programatically defined? Not a concern for CIViC (they def. all have labels) but not sure if we should require this.